aboutcitus/privacy.rst

```diff
@@ -0,0 +1,34 @@
+Privacy
+=======
+
+At Citus Data we know you trust our product to manage sensitive information. We take your privacy seriously, and want to be transparent about what information we collect.
+
+In order to monitor the usage of Citus across operating systems and data workload, each installation sends back anonymous statistics to our servers. You are able to opt out of this data collection.
+
+What we Collect
+---------------
+
+Reporting happens on installation and every twenty-four hours thereafter. Here is what happens:
+
+* Citus checks if there is a newer version of itself, and if so emits a notice to the database logs.
+* Citus collects and sends these statistics about your cluster:
+   * Randomly generated cluster identifier
+   * Number of servers
+   * OS and version (output of ``uname -psr`` command)
+   * Number of tables, rounded to a power of two
+   * Total size of shards, rounded to a power of two
+
```

Comment:
Also, whether Citus is running inside Docker or not.

More accurately, we send everything in the `pg_dist_node_metadata` table to the server. This table contains only one row/one column which is a `jsonb` field. By default, it only contains the random `server_id`, which is the cluster identifier you mentioned earlier. In citusdata/docker#50, we add a `docker` field to this value, so we also know if the collected report is from inside docker image or not.